xml/System.Windows.Forms/TextRenderer.xml

```diff
@@ -848,10 +868,10 @@
         <Parameter Name="font" Type="System.Drawing.Font" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <param name="font">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="text">The text to measure.</param>
+        <param name="font">The <see cref="T:System.Drawing.Font" /> to apply to the measured text.</param>
+        <summary>Provides the size, in pixels, of the specified text when drawn with the specified font.</summary>
+        <returns>The <see cref="T:System.Drawing.Size" />, in pixels, of text drawn on a single line with the specified font. You can manipulate how the text is drawn by using one of the <see cref="M:System.Windows.Forms.TextRenderer.DrawText(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},System.Drawing.Font,System.Drawing.Rectangle,System.Drawing.Color,System.Windows.Forms.TextFormatFlags)" /> overloads that takes a <see cref="T:System.Windows.Forms.TextFormatFlags" /> parameter. For example, the default behavior of the <see cref="T:System.Windows.Forms.TextRenderer" /> is to add padding to the bounding rectangle of the drawn text to accommodate overhanging glyphs. If you need to draw a line of text without these extra spaces you should use the versions of <see cref="M:System.Windows.Forms.TextRenderer.DrawText(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},System.Drawing.Font,System.Drawing.Point,System.Drawing.Color)" /> and <see cref="M:System.Windows.Forms.TextRenderer.MeasureText(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},System.Drawing.Font)" /> that take a Size and <see cref="T:System.Windows.Forms.TextFormatFlags" /> parameter. For an example, see <see cref="M:System.Windows.Forms.TextRenderer.MeasureText(System.Drawing.IDeviceContext,System.String,System.Drawing.Font,System.Drawing.Size,System.Windows.Forms.TextFormatFlags)" />.</returns>
```

Comment:
Maybe some of this information could be moved to the remarks. We probably don't need all this information to be clustered in the return value, we only need the essential description.